flytekit/tools/script_mode.py

```diff
@@ -127,6 +133,65 @@ def tar_strip_file_attributes(tar_info: tarfile.TarInfo) -> tarfile.TarInfo:
     return tar_info
 
 
+def add_imported_modules_from_source(source_path: str, destination: str, modules: List[ModuleType]):
```

Comment:
If `utils` imports another local module, will we copy that to the remote storage as well?
```
.
└── workflows
    ├── __init__.py
    ├── another_module
    │   └── wow.py
    ├── main.py  # only import util.py
    └── util.py # import util2.py
     └── util2.py     
```

Reply:
Any files that is a part of the import chain from the original workflow will be copied over.

Details: If `main.py` imports from `util.py` and `util.py` imports from `util2.py` then `util2.py` is part of `sys.modules` when `main.py` is loaded. With this PR, if the module is in `sys.modules` and in the same working directory as `main.py`, then it will be uploaded.

Reply:
nice, perfect